src/CMakeLists.txt

```diff
@@ -151,6 +171,21 @@ if (BUILD_MODULES)
     add_subdirectory(modules)
 endif()
 
+# Include External projects with tests
+if (BUILD_TELEMETRY)
```

Comment:
Remove this comment as well -- it does not stat anything useful, is clear that the telemetry project gets built.